jsapp/js/components/common/button.spec.tsx

```diff
@@ -4,55 +4,65 @@ import {render, screen} from '@testing-library/react';
 import {describe, it, expect, jest} from '@jest/globals';
 import userEvent from '@testing-library/user-event';
 
-describe('Button', () => {
-  it('Should render and be clickable when enabled', async () => {
-    const user = userEvent.setup();
+const user = userEvent.setup();
 
-    // Mock
-    const handleClickFunction = jest.fn();
+// Mock
+const handleClickFunction = jest.fn();
 
-    // Render
+
+describe('Enabled button', () => {
+  beforeEach(() => {
```

Comment:
let's move `handleClickFunction` into describe block to use as narrow scope as possible?

Reply:
Will do! 
I did it like that to avoid duplication, since the same function is used on both `describes` there. 
Since it's a simple function we can redeclare it, no problems, but in the cases where we can have a more complex mocked implementation it'd be good to have it declared and implemented only once!